docs/index.rst

```diff
@@ -1,19 +1,25 @@
-Welcome to mpl-interactions's documentation!
+mpl-interactions Documentation Home
 ===============================================
 
+Welcome to mpl-interactions' documentation!
+--------------------------------------------
+
+This library provides helpful ways to interact with `Matplotlib <http://www.matplotlib.org>`_ plots using three submodules. 
+Follow the links below for further information on installation, functions, and plot examples.
+
 .. toctree::
    :maxdepth: 2
 
    Installation
-   functions
+   Functions
```

Comment:
This name needs to be the same as the file `functions.rst` otherwise this will not be generated as a link:
![image](https://user-images.githubusercontent.com/10111092/89975163-0c897800-dc33-11ea-8733-246a2c3cb772.png). I think the best thing to do is to rename the `function` file to `Functions.rst`